mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ModClusterConfigResourceDefinition.java

```diff
@@ -65,10 +66,12 @@
  */
 class ModClusterConfigResourceDefinition extends SimpleResourceDefinition {
 
-    private static final String MOD_CLUSTER_SSL_CONTEXT_CAPABILITY_NAME = "org.wildfly.mod_cluster.ssl-context";
-    private static final RuntimeCapability<Void> MOD_CLUSTER_CAPABILITY = RuntimeCapability.Builder.of(MOD_CLUSTER_SSL_CONTEXT_CAPABILITY_NAME, false).build();
+    static final String LISTENER_CAPABILITY_NAME = "org.wildfly.undertow.listener";
+    private static final String MOD_CLUSTER_LISTENER_CAPABILITY_NAME = "org.wildfly.mod_cluster.listener";
+    private static final RuntimeCapability<Void> MOD_CLUSTER_LISTENER_CAPABILITY = RuntimeCapability.Builder.of(MOD_CLUSTER_LISTENER_CAPABILITY_NAME, false).build();
 
-    static final String SSL_CONTEXT_CAPABILITY_NAME = "org.wildfly.security.ssl-context";
+    private static final String MOD_CLUSTER_SSL_CONTEXT_CAPABILITY_NAME = "org.wildfly.mod_cluster.ssl-context";
+    private static final RuntimeCapability<Void> MOD_CLUSTER_SSL_CONTEXT_CAPABILITY = RuntimeCapability.Builder.of(MOD_CLUSTER_SSL_CONTEXT_CAPABILITY_NAME, false).build();
```

Comment:
Why two capabilities? Why named this way?

Does the user lose two independent things if the configured org.wildfly.undertow.listener and org.wildfly.security.ssl-context dependencies are not available? Or do they just lose "org.wildfly.mod_cluster"?